demo/ffi/cbits/flib.c

```diff
@@ -0,0 +1,18 @@
+#include "HsFFI.h"
+#include <stdio.h>
+#include <gdnative/gdnative.h>
+#include <pluginscript/godot_pluginscript.h>
+#include <gdnative_api_struct.gen.h>
```

Comment:
What's the reason for these includes?

Reply:
I had originally added deeper integration with the editor where there was syntax highlighting for .hs files but it wasn't a good experience. This is detritus from that. You're right, it can go.